src/lib/util/FuzzySearch.ts

```diff
@@ -1,6 +1,7 @@
-import { Collection, Message } from 'discord.js';
+import { Message } from 'discord.js';
 import { util } from 'klasa';
 import { levenshtein } from './External/levenshtein';
+import { Collection } from '@discordjs/collection';
```

Comment:
Why not import from discord.js?

Reply:
discord.js's Collection has `toJSON()`, which we don't need.

Reply:
Yes but for consistency in the code I would think using the discord.js one would be better. Surely having that method and not even calling it has no real overhead?

Reply:
It's unnecessary, that's why.

And as a general rule, I avoid all of discord.js's `toJSON` at all costs.

It gives all the data serialized plus bonus useless stuff.